README.md

```diff
@@ -1,66 +1,51 @@
+This is the root repository for the MOJ data catalogue.
 
-This template repository equips you with the default initial files required for a Ministry of Justice GitHub repository.
+## Component repositories
 
-## Included Files
+- [find-moj-data](https://github.com/ministryofjustice/find-moj-data/) - a read only view of the catalogue with GOV.UK styles
```

Comment:
```suggestion
- [find-moj-data](https://github.com/ministryofjustice/find-moj-data/) - a view of the catalogue with GOV.UK styles
```
we'll soon add registration of some kind (presumably under the find-moj-data brand?) and read only will be out of date